### ISSUE TITLE
Can't go back to 1.5.14 after trying the 2.0 (preview)

### ISSUE THREAD

**Issue by DylanComas** [digimezzo/dopamine-windows#1015]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report  
[ ] Feature request
</code></pre>

## I'm using these versions

* Windows version: 10 (1903)
* Dopamine version: 2.0 (Preview 12)

I just discovered Dopamine so I installed the 1.5.14 release which I thought was really good. Then I wanted to try the 2.0 preview but I got some lags so I tried to go back to the 1.5 but it just won't start. How do I fix that ?

EDIT: FIXED. There's some lingering files to delete even after a clean from CCleaner. They're located in the AppData folder !

**Comment by digimezzo** [comment]
@dexyel Indeed. The files in appdata need to be deleted, as the 1.5 version is not compatible with 2.0's database format.

Could you tell me more about the lags though? Version 2.0 is not supposed to be slower than 1.5 at this point. Thanks.

**Comment by DylanComas** [comment]
For me it is slower but maybe it's due to the size of my music folder. I had lags when scrolling down, when I went from the Collection to the Settings menu too. But the most annoying one was when I was using the cover feature. Some lags during the download with a spike everytime I clicked on OK. 
Also, it's not a lag but something I noticed, everytime you click on OK, the soft goes all the way up the library and it refreshes every cover of every album. It took me 45min on the 1.5 to check every cover of my library, and a bit more than 1h30 on the 2.0 :/

I hope it's clear enough, I'm not fluent in english yet ^^'
